src/main/java/com/conveyal/gtfs/model/Route.java

```diff
@@ -37,9 +63,56 @@ public class Route extends Entity { // implements Entity.Factory<Route>
     public int continuous_pickup = INT_MISSING;
     public int continuous_drop_off = INT_MISSING;
     public String network_id;
+    public String route_network_ids;
+
+    public static final String ROUTE_ID_FIELD = "route_id";
+    public static final String AGENCY_ID_FIELD = "agency_id";
+    public static final String ROUTE_SHORT_NAME_FIELD = "route_short_name";
+    public static final String ROUTE_LONG_NAME_FIELD = "route_long_name";
+    public static final String ROUTE_DESC_FIELD = "route_desc";
+    public static final String ROUTE_TYPE_FIELD = "route_type";
+    public static final String ROUTE_URL_FIELD = "route_url";
+    public static final String ROUTE_COLOR_FIELD = "route_color";
+    public static final String ROUTE_SORT_ORDER_FIELD = "route_sort_order";
+    public static final String ROUTE_TEXT_COLOR_FIELD = "route_text_color";
+    public static final String ROUTE_BRANDING_URL_FIELD = "route_branding_url";
+    public static final String CONTINUOUS_PICKUP_FIELD = "continuous_pickup";
+    public static final String CONTINUOUS_DROP_OFF_FIELD = "continuous_drop_off";
+    public static final String NETWORK_ID_FIELD = "network_id";
```

Comment:
Add a comment why field `network_id` is kept, in case someone wonders why it exists concurrently with `route_network_ids`.

Reply:
Fixed in: https://github.com/ibi-group/gtfs-lib/pull/16/commits/82684ac3a10245fd73a993a76471ab408648cb57